Tests/Editor/Picking/RectSelectionPicker.cs

```diff
@@ -0,0 +1,138 @@
+﻿using NUnit.Framework;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.ProBuilder;
+using UnityEngine.TestTools;
+using UObject = UnityEngine.Object;
+
+/// <summary>
+/// This class ensures we generate proper lookup textures when we need them during a marquee select.
+/// These textures must not render anything than our picking objects.
+/// The setup phase will create a set of objects put at the front of a camera. As there is no selectable object,
+/// we should only get white texture as a result.
+/// Issues will likely to appear if changes have been made in graphics pipeline.
+/// This set of tests should run against standard pipeline, URP and HDRP.
+/// </summary>
+class RectSelectionPicker
+{
+    ProBuilderMesh[] selectables;
+    GameObject[] sceneObjects;
+    Camera camera;
+
+    void Setup()
```

Comment:
Why is this not using [SetUp] test attribute?

Reply:
Because I followed the way it's being done in other test. I will change it.

Reply:
Fix in 085a884